gapic/api/artman_speech_v1_1beta1.yaml

```diff
@@ -0,0 +1,99 @@
+common:
+  api_name: speech
+  api_version: v1_1beta1
```

Comment:
IIRC this should be v1p1beta1

Reply:
Is this the thing that affects the endpoint? If so then yes.

Reply:
For reference: https://cloud.google.com/apis/design/versioning